polaris-react/playground/KitchenSink.tsx

```diff
@@ -2,51 +2,49 @@ import React from 'react';
 
 import {RenderPerformanceProfiler} from '../.storybook/RenderPerformanceProfiler';
 
-interface ReadmeModule {
-  [key: string]: any;
+interface Stories {
+  [key: string]: React.ComponentClass;
```

Comment:
`React.ComponentType` is more correct - as that encompases the typings for function based components and class based components

```suggestion
  [key: string]: React.ComponentType;
```

Reply:
Ahh love this. I had something there wasn't confident so I removed it. This is exactly it, cheers @BPScott .